networkx/algorithms/cycles.py

```diff
@@ -50,7 +50,7 @@ def cycle_basis(G, root=None):
     >>> G = nx.Graph()
     >>> nx.add_cycle(G, [0, 1, 2, 3])
     >>> nx.add_cycle(G, [0, 3, 4, 5])
-    >>> print(nx.cycle_basis(G, 0))
+    >>> nx.cycle_basis(G, 0)
```

Comment:
If the ordering of cycle bases can change (i.e. the result could be either `[[1, 2, 3, 0], [3, 4, 5, 0]]` or `[[3, 4, 5, 0], [1, 2, 3, 0]]`), then we might want to add `sorted()` here so that the doctests are always deterministic. If, however, the ordering is *already* deterministic (I think this is the case, just want to double check), then there's no need!